pkg/capabilities/consensus/types/aggregator.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+import (
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type Aggregator interface {
```

Comment:
Why do we need a separate types package for this? The aggregator should be an internal, private implementation detail and we shouldn't have to expose it elsewhere (either as a public interface, or in a types package).

Reply:
Ah, I guess your intent is to implement each aggregator in a separate package? Hmmm... wonder if that's overkill? IMO far simpler to just put everything in one package which gives us the flexibility to play around with package boundaries, and move things out a later date if that makes sense.

Reply:
Yeah, it's either everything in a single package or everything separated, otherwise we will run into circular dependencies (e.g. when we add a factory method). My slight preference is to keep stuff separated. Any particular worries about extra packages?

Reply:
Keeping separate for now.